src/main/java/ch/njol/skript/expressions/ExprColorOf.java

```diff
@@ -57,29 +59,29 @@
 public class ExprColorOf extends PropertyExpression<Object, Color> {
 
 	static {
-		register(ExprColorOf.class, Color.class, "colo[u]r[s]", "blocks/itemtypes/entities/fireworkeffects");
+		register(ExprColorOf.class, Color.class, "colo[u]r[s]", "blocks/itemtypes/entities/fireworkeffects/bossbars");
 	}
-	
+
 	@SuppressWarnings("null")
 	@Override
```

Comment:
```suggestion
	@Override
```
And on the other methods as well